candidate_models/base_models/__init__.py

```diff
@@ -255,8 +255,7 @@ def voneresnet(model_name='resnet50'):
     from model_tools.activations.pytorch import load_preprocess_images
     preprocessing = functools.partial(load_preprocess_images, image_size=224,
                                       normalize_mean=(0.5, 0.5, 0.5), normalize_std=(0.5, 0.5, 0.5))
-    from candidate_models.base_models.stochastic import StochasticPytorchWrapper
-    wrapper = StochasticPytorchWrapper(identifier='vone'+model_name, model=model, preprocessing=preprocessing)
+    wrapper = PytorchWrapper(identifier='vone'+model_name, model=model, preprocessing=preprocessing)
```

Comment:
I'm a bit confused by this -- since the VOneNets defined here no longer use anything stochastic, is it even necessary to have a separate pool for them? (https://github.com/brain-score/candidate_models/pull/70/files#diff-30b56a69b0b5a99bcaf36381fc10683e55aff0b7a42516cd0ee8f6d1ff9c12a0R498)